dashboard/d_graphs.py

```diff
@@ -164,7 +171,8 @@ def add_topic_rows(df: pd.DataFrame) -> str:
         for score in row:
             if isinstance(score, float):
                 color = "#fabbb7" if score < -0.5 else "#b6f7ae" if score > 0.5 else "#fafafa"
```

Comment:
Fantastic idea to change the colour based on score, that will make the graphs look great!